assets/js/admin/marketplace-suggestions.js

```diff
@@ -0,0 +1,340 @@
+/* global marketplace_suggestions ajaxurl */
+( function( $, marketplace_suggestions ) {
+	$( function() {
+		if ( 'undefined' === typeof marketplace_suggestions ) {
+			return;
+		}
+
+		// Stand-in wcSettings.recordEvent for until tracks PR is merged.
+		// https://github.com/woocommerce/woocommerce/pull/22746
+		window.wcSettings = window.wcSettings || {};
+		window.wcSettings.recordEvent = window.wcSettings.recordEvent  || function() { };
+
+		// Tracks events sent in this file:
+		// - marketplace_suggestion_displayed
+		// - marketplace_suggestion_clicked
+		// - marketplace_suggestion_dismissed
+		// All are prefixed by {WC_Tracks::PREFIX}.
+		// All have one property for `suggestionSlug`, to identify the specific suggestion message.
+
+		function dismissSuggestion( suggestionSlug ) {
+			// hide the suggestion in the UI
+			var selector = '[data-suggestion-slug=' + suggestionSlug + ']';
+			$( selector ).fadeOut();
```

Comment:
Remove after fade out otherwise the zebra striping is broken once dismissed.

Reply:
👍  

https://github.com/woocommerce/woocommerce/pull/22712/commits/e37583bb2b1dc43e54b02ebf52eb34a81f0d709e